NON_BITGO_RECOVERY.md

```diff
@@ -0,0 +1,44 @@
+# Non BitGo Recovery Instructions
+
+Build a transaction using the recovery KeyCard, independent from any BitGo services.
+
+## What happens in a Non-BitGo recovery?
+
+1. You are asked to provide the Coin Name that you want to recover.
+
+2. You are asked to provide the Key Recovery Service. This is the key recovery service that you choose to manage your backup key. If you have the encrypted backup key, you may leave this blank.
+There are 3 options Keyternal, BitGo KRS, and Coincover. Keyternal and BitGo KRS support BTC and ETH only. Coincover supports BTC, ETH, XLM, XRP, DASH, ZEC, LTC, BCH, BSV.
+
+3. You are asked to provide the Environment (Testnet/Mainnet).
+
+4. You are asked to provide the Box A Value. This is your encrypted user key as found on your recovery KeyCard.
```

Comment:
I guess in the case of self managed cold wallets, this flow does not apply?

Reply:
That's correct, in case of self managed cold wallet (unsigned sweep), information about ENV is not taken and public key of user key is taken

Reply:
Ok, lets mention it does not apply in this case

Reply:
I agree, the steps that you provide apply only for a Hot Wallet Keycard.

Reply:
Added that it is valid only for hot wallets